frontend/src/modules/EmibInboxRedux.js

```diff
@@ -46,6 +47,12 @@ const updateTask = (emailIndex, responseId, taskAction) => ({
   responseId,
   taskAction
 });
+// emailIndex refers to the index of the original parent email and responseId is the id of the response that is being deleted
+const deleteEmail = (emailIndex, responseId, emailAction) => ({
```

Comment:
Here, you have an unused parameter: _emailAction_

Reply:
resolved in c175f25